x/msgfees/keeper/params.go

```diff
@@ -9,8 +9,9 @@ import (
 // GetParams returns the total set of distribution parameters.
 func (k Keeper) GetParams(ctx sdk.Context) (params types.Params) {
 	return types.Params{
-		FloorGasPrice:  k.GetFloorGasPrice(ctx),
-		NhashPerUsdMil: k.GetNhashPerUsdMil(ctx),
+		FloorGasPrice:      k.GetFloorGasPrice(ctx),
+		NhashPerUsdMil:     k.GetNhashPerUsdMil(ctx),
+		ConversionFeeDenom: k.GetConversionFeeDenom(ctx),
 	}
 }
```

Comment:
lgtm but we should modify the upgrades.go changes on this one too
```
"ochre-rc1": { // upgrade for 1.13.0-rc1
```
Unless you think a different PR for that is better?

Reply:
I think we need to cherry pick this and to the release branch then add add the upgrade handler.

Reply:
I was thinking a different PR at first, but on second thought...  I think it makes sense to add it here.